src/io.rs

```diff
@@ -204,7 +204,7 @@ pub fn load(vm: &Thread) -> Result<()> {
     vm.add_bytecode("io_bind", io_bind_type, 3, io_bind);
```

Comment:
I would like to rename this to `io_flat_map`, but it wasn't very clear to me how to rewrite it with the arguments swapped. Still haven't got my head around the VM's bytecode! :o